Packs/DSPM/Integrations/DSPM/DSPM.yml

```
@@ -0,0 +1,984 @@
+      secret: false
+    - description: 'List of policy names.'
+      isArray: true
+      name: policy_name_in
+      required: false
+      secret: false
+    - description: 'Exact policy name.'
+      isArray: false
+      name: policy_name_equals
+      required: false
+      secret: false
+    - description: 'List of asset names.'
+      isArray: true
+      name: asset_name_in
+      required: false
+      secret: false
+    - description: 'Exact asset name.'
+      isArray: false
+      name: asset_name_equals
+      required: false
+      secret: false
+    - description: 'List of cloud providers ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"].'
+      isArray: true
+      name: cloud_provider_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'Exact cloud provider ["AWS", "AZURE", "GCP", "SNOWFLAKE", "FILE_SHARE", "O365"].'
+      isArray: false
+      name: cloud_provider_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'AWS'
+      - 'AZURE'
+      - 'GCP'
+      - 'SNOWFLAKE'
+      - 'FILE_SHARE'
+      - 'O365'
+    - description: 'List of destination project vendor names.'
+      isArray: true
+      name: destination_project_vendor_name_in
+      required: false
+      secret: false
+    - description: 'Exact destination project vendor name.'
+      isArray: false
+      name: destination_project_vendor_name_equals
+      required: false
+      secret: false
+    - description: 'List of cloud environments ["UNKNOWN", "DEVELOPMENT", "STAGING", "TESTING", "PRODUCTION"].'
+      isArray: true
+      name: cloud_environment_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'UNKNOWN'
+      - 'DEVELOPMENT'
+      - 'STAGING'
+      - 'TESTING'
+      - 'PRODUCTION'
+    - description: 'Exact cloud environment ["UNKNOWN", "DEVELOPMENT", "STAGING", "TESTING", "PRODUCTION"].'
+      isArray: false
+      name: cloud_environment_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'UNKNOWN'
+      - 'DEVELOPMENT'
+      - 'STAGING'
+      - 'TESTING'
+      - 'PRODUCTION'
+    - description: 'List of policy severities ["HIGH", "MEDIUM", "LOW"].'
+      isArray: true
+      name: policy_severity_in
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'HIGH'
+      - 'MEDIUM'
+      - 'LOW'
+    - description: 'Exact policy severity ["HIGH", "MEDIUM", "LOW"].'
+      isArray: false
+      name: policy_severity_equals
+      required: false
+      secret: false
+      auto: PREDEFINED
+      predefined:
+      - 'HIGH'
+      - 'MEDIUM'
+      - 'LOW'
+    - description: 'List of category types ["FIRST_MOVE", "ATTACK", "COMPLIANCE", "ASSET_AT_RISK", "RECONNAISSANCE"].'
```

```suggestion
    - description: 'List of category types.'
```
Is this a comma-separated list?  If so, it should be written as:
A comma-separated list of category types.